src/components/ebay-menu-button/examples/22-badged-items/template.marko

```diff
@@ -0,0 +1,5 @@
+<ebay-menu-button text="eBay Menu">
+    <@item badge-number=1>item 1 that has very long text</@item>
+    <@item badge-number=100>item 2</@item>
```

Comment:
I don't think we go beyond 2 digits. It caps out at `99+`.

Reply:
that's correct, I just wanted to display the `99+` so I put `100`. I can change it to 99 if you want.

Reply:
Yes please